types/src/keyless/circuit_constants.rs

```diff
@@ -92,3 +97,21 @@ pub fn devnet_prepared_vk() -> PreparedVerifyingKey<Bn254> {
 
     PreparedVerifyingKey::from(vk)
 }
+
+pub struct Groth16Keys {
+    pub pk: Trapdoor<Bn254>,
+    pub vk: VerifyingKey<Bn254>,
+    pub prepared_vk: PreparedVerifyingKey<Bn254>,
```

Comment:
Seems odd to keep both the vk and pvk in the same struct? I don't see why you'd need the vk after computing the pvk